src/Symfony/Component/Console/Attribute/Reflection/ReflectionMember.php

```diff
@@ -96,4 +96,9 @@ public function getMemberName(): string
     {
         return $this->member instanceof \ReflectionParameter ? 'parameter' : 'property';
     }
+
+    public function isBackedEnumType(): bool
+    {
+        return $this->member->getType() instanceof \ReflectionNamedType && is_subclass_of($this->member->getType()->getName(), \BackedEnum::class);
```

Comment:
Most reflection classes have public properties that we use when possible.
```suggestion
        return $this->member->getType() instanceof \ReflectionNamedType && is_subclass_of($this->member->getType()->name, \BackedEnum::class);
```

Reply:
I couldn't find a public property on `\ReflectionNamedType`, am I overlooking something?

Reply:
Indeed.